introduction/0X_gallery.rst

```diff
@@ -7,15 +7,19 @@ with A+.
 Front page
 ----------
 
-.. figure:: /images/gallery/aplus-front-page.jpg
-
-The above image shows a screenshot of the front page of `A+ at Aalto University
-<https://plus.cs.aalto.fi/>`_ from Summer 2019. The front page shows at most
-eight recent courses. Each course has optional logo image, name, course
-code and teaching time. All courses have a green "Aalto" tag to indicate that
-the course is for Aalto University students. The Ohjelmointistudio 2 course
-has also a blue "MOOC" tag to indicate the course is also a massive open online
-course. Sections `7.1 Settings a course on production servers
+.. figure:: /images/gallery/aplus-front-page.png
+   :alt: Front page of A+
+
+The above image shows an example of the front page of `A+ at Aalto University
+<https://plus.cs.aalto.fi/>`_. The front page has two sections. The upper section shows
+course instances which the logged-in user is currently; enrolled in, teaching
+or assisting. The lower section shows all ongoing and recently ended courses.
+
+Each course has optional image, name, instance name, course code and teaching schedule. 
+All courses have a green "Aalto" tag to indicate that the course is for Aalto University students. 
+The "Data Structures and Algorithms" -course has also a blue "MOOC" tag to indicate
+that the course is also a massive open online course. 
+Sections `7.1 Settings a course on production servers
 <../m07_admin/01_setup>`_
 and `7.2 Course settings <../m07_admin/02_settings>`_ describe these in detail.
```

Comment:
Now, that you are updating this specific page. Could you fix the internal references?
```suggestion
and :doc:`7.2 Course settings</admin/02_settings>` describe these in detail.
```

Reply:
Done